### ISSUE TITLE
why every one is forking ?

### ISSUE THREAD

**Issue by Neeraj81** [mapup/MapUp-Data-Assessment-F#91]
why every person is forking this repo 
instead of cloning/copy/download the repo

they said in the form that you need to create a clone (which is not fork) and make it private (and add contributors). then add your submitting into this clone, rather than forking it and committing it !

im i wrong or am i right?

**Comment by Neeraj81** [comment]
dont fork and commit guys, 

rather copy/download/clone/import and then make that repo private and add the submission folder and submit the link in the gform.

**Comment by HiteshAroraCool** [comment]
you are right! unfortunately, some of us(me) didn't know the difference between clone and fork.

**Comment by Neeraj81** [comment]
> dont fork and commit guys,
> 
> rather copy/download/clone/import and then make that repo private and add the submission folder and submit the link in the gform.

@Venkatesh1618 @saransh-mapup Can you confirm this once?

**Comment by neha03109** [comment]
@Neeraj81 

Please correct me , I am sure about what I did.  I didnt fork the repo. I created the private  new repo and import the repo and added collaborators name.
Then using the git command created folder and added files then push the changes. In the form. I have mentioned my rep link

**Comment by prime1289** [comment]
> why every person is forking this repo instead of cloning/copy/download the repo
> 
> they said in the form that you need to create a clone (which is not fork) and make it private (and add contributors). then add your submission into this clone, rather than forking it and committing it !
> 
> im i wrong or am i right?

You are right
